tmol/utility/cpp_extension.py

```diff
@@ -21,8 +22,12 @@
 _default_include_paths = list(tmol_include_paths() + extern_include_paths())
 
 _required_flags = ["--std=c++14", "-DWITH_NVTX", "-w"]
-_default_flags = ["-O3"]
-# _default_flags = ["-g", "-Og", "-DDEBUG"]
+
+if os.environ.get("DEBUG"):
+    _default_flags = ["-O3", "-DDEBUG"]
```

Comment:
do you also want to enable this on test server?  if so I think you need to change file in .buildkite

Reply:
Right!